xla/service/gpu/gpu_fusible.cc

```diff
@@ -858,6 +858,7 @@ std::vector<HloComputation*> GetFusibleComputations(
       // flow. See also fusion_wrapper.cc, which does the same.
       if (HloInstruction::MightHaveCalledComputations(instr->opcode()) &&
           instr->opcode() != HloOpcode::kWhile &&
+          instr->opcode() != HloOpcode::kCall &&
```

Comment:
I'm not sure we want to fuse inside all calls. We might want to just enable the async ones you care about for now.

Reply:
I broke out this check, now it only applies to calls with stream annotations.